templates/miq-template-ext-db.yaml

```diff
@@ -107,7 +107,7 @@ objects:
       spec:
         containers:
         - name: manageiq
```

Comment:
Do we also want to change this dc and container to be named `manageiq-frontend`?